templates/education-internships.html

```diff
@@ -15,18 +15,63 @@ <h3>Internships</h3>
 
 <div>
 
+        <div class="text-center">
+            <figure><img src="{{ url_for('static', filename='img/HLT_Footer.jpeg') }}" alt="HLT Banner" style="width: 100%;"></figure>
+        </div>
+
     <p>
-        ...
+        The Chair in Computational Lexicology liaises with a range of companies in the Netherlands and abroad.
+        This involves cooperation projects, internships, participation in minor programs for employees, advice
+        and consultation. Recent contacts are: <a href="https://www.accenture.com/nl-en" target="_blank">Accenture</a>,
```

Comment:
maybe change "Recent contacts are:" to "Recent contacts are, among others:" or "Some recent contacts are:" as the list is not complete (and may never be)